src/Compilers/CSharp/Portable/Symbols/SpecialTypeExtensions.cs

```diff
@@ -27,20 +27,6 @@ public static bool IsIntegralType(this SpecialType specialType)
             }
         }
 
-        public static bool IsSignedIntegralType(this SpecialType specialType)
```

Comment:
>IsSignedIntegralType [](start = 27, length = 20)

Was this method unused?

Reply:
Correct, the method was unused.

---
In reply to: [392394425](https://github.com/dotnet/roslyn/pull/42360#discussion_r392394425) [](ancestors = 392394425)